Fix #18544 : Added a new display style to the parent container of the Voice artist warning message to address the bug

scoopstroop92 opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes #18544.
2. This PR does the following: Makes the  parent container of the Voice artist warning message a flex  
3. (For bug-fixing PRs only) The original bug occurred because: The parent container of the component under fix doesnt scale as expected  for smaller screen size whereas the the design and layout of other cards in the settings  page of the contributor dashboard are styled  to display as flex  


## Essential Checklist

- [x] The **PR title** starts with "Fix #bugnum: " or "Fix part of #bugnum: ...",
followed by a short, clear summary of the changes.
- [x] I have followed the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Make-a-pull-request).
- [x] The linter/Karma presubmit checks have passed on my local machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] I have assigned the correct reviewers to this PR (or will leave a
comment with the phrase "@{{reviewer_username}} PTAL" if I don't have
permissions to assign reviewers directly).




## Proof that changes are correct

Environment : Local development server
Device : Computer 
Browser :  Chrome 
![image](https://github.com/oppia/oppia/assets/89441435/4fa95f48-972b-43a7-8dd1-8ff32303eb7f)

### Console Screenshot 👍 
![image](https://github.com/oppia/oppia/assets/89441435/50eaac8a-820f-4cfa-8011-566b210eeb74)


<!--
Add videos/screenshots of the user-facing interface to demonstrate that the changes
made in this PR work correctly. If this PR is for a developer-facing feature,
provide the videos/screenshots of developer-facing interface.

Please also include videos/screenshots of the developer tools
browser console, so that we can be sure that there are no console errors.

If the changes in your PRs are autogenerated via a script and you cannot provide proof
for the changes then please leave a comment "No proof of changes needed because {{Reason}}"
and remove all the sections below.
-->

#### Proof of changes on desktop with slow/throttled network
![image](https://github.com/oppia/oppia/assets/89441435/cfb26f45-7105-4aa1-9490-dc44ec7936d1)

<!--
Make sure to properly verify that everything works correctly, and that there are
no weird UI mistakes or other problems. Also, if there are any newly added fields,
try to fill them out and test that different inputs are correctly accepted/rejected.

Throttle the network (to 3G) using the browser Developer Tools (see references below).
There should be no performance or UI issues while the network is slow.

References:
 - Chrome: https://css-tricks.com/throttling-the-network/
 - Firefox: https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor/Throttling
-->

#### Proof of changes on mobile phone
Used the Developer Tools emulator in chrome for this.

![image](https://github.com/oppia/oppia/assets/89441435/50309397-c627-4ed0-854b-aa78654c076b)


<!--
In some cases this is not needed (e.g. for pages that we do not expect to
support mobile phones, or for backend-only features).

Feel free to use the Developer Tools emulator for this.

References:
 - Chrome: https://developer.chrome.com/docs/devtools/device-mode/
 - Firefox: https://firefox-source-docs.mozilla.org/devtools-user/index.html#responsive-design-mode
-->


<!--
If the PR changes the UI, make sure to add screenshots with the site
language set to Arabic as well (we use Arabic as it is a language written from right to left).
-->

## PR Pointers

- Never force push! If you do, your PR will be closed.
- Make sure your PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays.
- Some e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-CI-checks-fail-on-your-PR).
- See the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia's-code-owners-and-checks-to-be-carried-out-by-developers) for what code owners will expect.

Commit:
Initial commit for local testing

Commit:
Initial commit for local testing

Commit:
making the parent container flex instead of the warning message

Commit:
making the parent container flex instead of the warning message

Commit:
The fix addresses the Voice artists warning message in creatordashboard settings

Commit:
The fix addresses the Voice artists warning message in creatordashboard settings

scoopstroop92 commented:
@seanlip @Uglycrown @SanjaySajuJacob @Priyansh61 @KolliAnitha PTAL . I have updated it in  the description as well

prafulbbandre commented:
Hi @ramyaram2092 Please follow the first point of essential checklist and link this pr to the issue.

scoopstroop92 commented:
@prafulbbandre  I linked it with the issue. Since this is my first PR could you help confirm if I have added all details that are required . Just wanted to make sure . Thank you

prafulbbandre commented:
> @prafulbbandre I linked it with the issue. Since this is my first PR could you help confirm if I have added all details that are required . Just wanted to make sure . Thank you

@ramyaram2092 FYI : You had not linked this PR to any issue but you can do that by deleting this [or fixes part of] part from the first point of overview, if you did this thing you will see that the issue will link to the pr and you can confirm it by looking at the develoment section in the right side bar.

Just delete the reviewers section.
Other than that your description looks good.

One thing why do you assign so many people do you need to confirm changes from them or just need to merge your Pr.

prafulbbandre commented:
@ramyaram2092 If you just need to merge your Pr then assign the reviewers that are assign to your pr for review you can see them in the top of the right side bar and can assign them by using @reviewername PTAL

scoopstroop92 commented:
> > @prafulbbandre I linked it with the issue. Since this is my first PR could you help confirm if I have added all details that are required . Just wanted to make sure . Thank you
> 
> @ramyaram2092 FYI : You had not linked this PR to any issue but you can do that by deleting this [or fixes part of] part from the first point of overview, if you did this thing you will see that the issue will link to the pr and you can confirm it by looking at the develoment section in the right side bar.
> 
> Just delete the reviewers section. Other than that your description looks good.
> 
> One thing why do you assign so many people do you need to confirm changes from them or just need to merge your Pr.

I am new to oppia. I think in the code contribution  guideline for first time contributors it is mentioned to include the LACE team if working on an issue from the LACE project.  So I include the Lace team and contributors who are part of the comment thread

prafulbbandre commented:
> > > @prafulbbandre I linked it with the issue. Since this is my first PR could you help confirm if I have added all details that are required . Just wanted to make sure . Thank you
> > 
> > 
> > @ramyaram2092 FYI : You had not linked this PR to any issue but you can do that by deleting this [or fixes part of] part from the first point of overview, if you did this thing you will see that the issue will link to the pr and you can confirm it by looking at the develoment section in the right side bar.
> > Just delete the reviewers section. Other than that your description looks good.
> > One thing why do you assign so many people do you need to confirm changes from them or just need to merge your Pr.
> 
> I am new to oppia. I think in the code contribution guideline for first time contributors it is mentioned to include the LACE team if working on an issue from the LACE project. So I include the Lace team and contributors who are part of the comment thread

Please let me know if you just want to merge your pr.

prafulbbandre commented:
> > > @prafulbbandre I linked it with the issue. Since this is my first PR could you help confirm if I have added all details that are required . Just wanted to make sure . Thank you
> > 
> > 
> > @ramyaram2092 FYI : You had not linked this PR to any issue but you can do that by deleting this [or fixes part of] part from the first point of overview, if you did this thing you will see that the issue will link to the pr and you can confirm it by looking at the develoment section in the right side bar.
> > Just delete the reviewers section. Other than that your description looks good.
> > One thing why do you assign so many people do you need to confirm changes from them or just need to merge your Pr.
> 
> I am new to oppia. I think in the code contribution guideline for first time contributors it is mentioned to include the LACE team if working on an issue from the LACE project. So I include the Lace team and contributors who are part of the comment thread

the persons who owns the file you changed will be assign to your PR by the oppiabot as reviewers, you need to assign them to your PR for review using @reviewwername PTAL they will suggest changes if there are some other wise they will merge the pr.

scoopstroop92 commented:
> > > > @prafulbbandre I linked it with the issue. Since this is my first PR could you help confirm if I have added all details that are required . Just wanted to make sure . Thank you
> > > 
> > > 
> > > @ramyaram2092 FYI : You had not linked this PR to any issue but you can do that by deleting this [or fixes part of] part from the first point of overview, if you did this thing you will see that the issue will link to the pr and you can confirm it by looking at the develoment section in the right side bar.
> > > Just delete the reviewers section. Other than that your description looks good.
> > > One thing why do you assign so many people do you need to confirm changes from them or just need to merge your Pr.
> > 
> > 
> > I am new to oppia. I think in the code contribution guideline for first time contributors it is mentioned to include the LACE team if working on an issue from the LACE project. So I include the Lace team and contributors who are part of the comment thread
> 
> the persons who owns the file you changed will be assign to your PR by the oppiabot as reviewers, you need to assign them to your PR for review using @reviewwername PTAL they will suggest changes if there are some other wise they will merge the pr.

Makes sense. Going forward will do the same

prafulbbandre commented:
@ramyaram2092 which editor have you used, did you format the file.

scoopstroop92 commented:
> @ramyaram2092 which editor have you used, do you format the file.

I use VS code

prafulbbandre commented:
> > @ramyaram2092 which editor have you used, do you format the file.
> 
> I use VS code

you have changed the format of the file please make it as it was in the first place this is causing the link checks to fail.

scoopstroop92 commented:
> > > @ramyaram2092 which editor have you used, do you format the file.
> > 
> > 
> > I use VS code
> 
> you have changed the format of the file please make it as it was in the first place this is causing the link checks to fail.

Yea I see that. Let me check.

Commit:
Removed the extra space added by vscode autoformat

Commit:
Removed the extra space added by vscode autoformat

scoopstroop92 commented:
@prafulbbandre : VsCode was autoformatting everytime I saved a file. I disabled that option and saved the file originally as it was.  Ran the Lint Check locally after making the changes. Now the  Lint Checks e2e test cases should pass. Please advise if I should create a new PR

![image](https://github.com/oppia/oppia/assets/89441435/d690d8e7-c3dc-4953-8861-417d184ee82f)

lkbhitesh07 committed:
Edit default value for feature flag and platform parameter (#18413)

* initial changes

* fixed tests

* fixed e2e test

* fixed lint

* fixed the failing tests

* ts check fix

* fixed frontend tests

* updates to the features tab

* bit

* changed feature stage of dummy-feature

* fixed backend test

* added blank line to the spec file

* change the default value of feature flag

* added the default_value field to the model

* added for feature flags

* handled the incoming schema for feature flags

* fixed backend tests

* fixed tests

* fixed backend tests

* fixed tests

* fixed lint

* fixed single mypy

* fixed mypy tests

* more mypy

* added button to update the default values

* fixed lint

* more lint

* nit

* return default value from parameter registry when not present in storage model

* added test

* fixed lint

* fixed mypy

---------

Co-authored-by: lkbhitesh07 <lkbhiteh07@gmail.com>

Ykumar1415 committed:
Fix #18333 [Feature request]:Fixed The Problem Of Limited Text Formatting #18333  (#18397)

* limited text formatting error fixed

* Update blog-dashboard-page.component.html

* Update goals-tab.component.ts

* Update goals-tab.component.ts

gupta-arpan committed:
Fix #18540: Update modal labels to Publish Changes (#18570)

Update modal labels to Publish Changes

Co-authored-by: Arpan Gupta <gupta-arpan@pop-os.localdomain>

seanlip committed:
Complete some miscellaneous small fixes -- drop UA tag, add comments and update issue templates. (#18553)

* Misc small fixes -- update Android page, add comments and update issue templates, drop UA tag.

* Drop changes for updating Android page description.

Patel-Muhammad committed:
Contributor dashboard -- Fixes axeDev tools + lighthouse error and makes the lighthouse score to 1 (#18454)

added a11y fixes

Patel-Muhammad committed:
Added i18n for user facing texts in the question player and story revision page (#18491)

* added i18n

* added i18n in qqq.json

* fixed tests

prafulbbandre committed:
Fix #18153: Submit button text is visible properly. (#18485)

* fixed state of submit button

* fixed state of submit button

* fixed lint

thesatyam05 committed:
issue #17997 Fixed Misaligned Text in blog-home-page.component.html ( synced pull ) (#18547)

Updated branch for pull req : #18472 and issue: #17997

Fixed the misalignment of the text "Sorry, there are no blog posts to show." in file blog-home-page.component.html

Adarsh7825 committed:
Fix auto-hover issue in Community Lessons section (#18451)

* make changes in oppia.css file

* Fixed the auto-hover

---------

Co-authored-by: Kshitij Patil <91470808+kshitij01042002@users.noreply.github.com>

prafulbbandre commented:
@ramyaram2092 Provide the screenshot of the changes you made on the iphone SE dimensions.
As the issue is observed on the small screen device.

scoopstroop92 commented:
@prafulbbandre  I tested for all screen size in the emulator .
iphone SE
![image](https://github.com/oppia/oppia/assets/89441435/c02a07b0-b029-48ea-99af-751034d184b3)

Commit:
Merge branch 'creatordashboard-voiceartist-fix' of https://github.com/ramyaram2092/oppia into local branch creatordashboard-voiceartist-fix

scoopstroop92 commented:
> Hi @ramyaram2092, the build of this PR is stale and this could result in tests failing in develop. Please update this pull request with the latest changes from develop. Thanks!

Done

prafulbbandre commented:
> > Hi @ramyaram2092, the build of this PR is stale and this could result in tests failing in develop. Please update this pull request with the latest changes from develop. Thanks!
> 
> Done

https://github.com/oppia/oppia/pull/18572#issuecomment-1617627375 same here

kevintab95 commented:
There seems to be changes unrelated to the original issue in this PR (similar to https://github.com/oppia/oppia/pull/18572#pullrequestreview-1511415127). Please fix this and assign me once it is ready for a review. Thanks!

seanlip commented:
I think this PR needs to be redone -- the changes shown in https://github.com/oppia/oppia/pull/18571/files are not the right ones to fix the bug. 

@ramyaram2092 Please open a fresh PR and check that the "Files Changed" tab changes are correct before requesting reviews. Thanks!

scoopstroop92 commented:
Since the branch became stale I had to repull the changes from develop that's when the issue occurred. I did rebase. How do you merge latest changes from develop if not for rebase. Kindly advise .
I will create a new PR by EOD (PST) @seanlip @kevintab95

U8NWXD commented:
@ramyaram2092 `merge` and `rebase` are two different git commands. You need to use `merge` instead of `rebase`. Here's an explanation of the differences: https://www.atlassian.com/git/tutorials/merging-vs-rebasing
